docs/en/basic/Configuration-Settings.md

```diff
@@ -139,15 +139,34 @@ master's `alluxio-site.properties` file.
 After restarting the cluster, all jobs will automatically set `alluxio.user.file.writetype.default`
 to `CACHE_THROUGH`.
 
-Clients can ignore or overwrite the cluster-wide default values by either specifying the user property
-`alluxio.user.conf.cluster.default.enabled=false` to decline loading the cluster-wide default values
-or following the approaches described in [Configure Alluxio for Applications](#configure-applications)
-to overwrite the same properties.
+Clients can ignore or overwrite the cluster-wide default values by following the approaches described in
+[Configure Alluxio for Applications](#configure-applications) to overwrite the same properties.
 
 > Note that, before version 1.8, `${ALLUXIO_HOME}/conf/alluxio-site.properties` file is only loaded
 by Alluxio server processes and will be ignored by applications interacting with Alluxio service
 through Alluxio client, unless `${ALLUXIO_HOME}/conf` is on applications' classpath.
 
+### Path Defaults
+
+Since version 2.0, Alluxio administrator can set default client side configurations for Alluxio paths.
+FileSystem client operations have options, by default, the options are derived from client side configuration
```

Comment:
remove `by default`? 
`FileSystem client operations have options that are derived from client side configuration`